maint-lib/stage.py

```diff
@@ -45,10 +45,26 @@
     sys.exit(1)
 
 
+def populateCephPointRelease():
```

Comment:
Move to `envglobals.py` with the other computed variables. rename to `exportCephVersionSpecifics()`.

Reply:
Ok